code/modules/clothing/outfits/factions/syndicate.dm

```diff
@@ -100,6 +100,35 @@
 
 	assign_codename(H)
 
+/datum/outfit/job/syndicate/assistant/suns
+	name = "Syndicate - Freshman (SUNS)"
+
+	uniform = /obj/item/clothing/under/syndicate/suns
+	alt_uniform = /obj/item/clothing/under/syndicate/suns/alt
+	suit = /obj/item/clothing/suit/toggle/suns/alt
+	alt_suit = /obj/item/clothing/suit/toggle/suns
+	shoes = /obj/item/clothing/shoes/laceup/suns
+	gloves = null
+	ears = null
+	box = /obj/item/storage/box/survival
+
+	backpack = /obj/item/storage/backpack
+	satchel  = /obj/item/storage/backpack/satchel
+	duffelbag = /obj/item/storage/backpack/duffelbag
```

Comment:
I don't think these have to be defined, they should be on base types. (the backpacks)

Reply:
The base types for the Syndicate are, in fact, security. This change was requested by meem.

Reply:
right